<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>999</width>
    <height>736</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="my_label" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>700</width>
      <height>700</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="mouse_movement">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>70</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>50</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>    Mouse Movement</string>
    </property>
   </widget>
   <widget class="QLabel" name="mouse_pressed">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>70</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>50</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Mouse Pressed</string>
    </property>
   </widget>
   <widget class="Line" name="x_axis">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>225</y>
      <width>700</width>
      <height>1</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 85, 255);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="y_axis">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>-70</y>
      <width>1</width>
      <height>700</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(85, 255, 255);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="show_axes">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>10</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Axes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>110</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>RESET</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>160</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Create Grid</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>160</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>160</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Create Axis</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>190</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Parametric </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>190</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>DDA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Bresenham_line">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>190</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Bresenham</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>230</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Execution TIme</string>
    </property>
   </widget>
   <widget class="QLabel" name="execution_time">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>220</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="cir_polar">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>300</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Circle Polar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>270</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Radius</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="Radius">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>270</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="value">
     <number>5</number>
    </property>
   </widget>
   <widget class="QPushButton" name="cir_midpoint">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>330</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Circle Midpoint</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>390</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>r1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>390</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>r2</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="r1">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>390</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="value">
     <number>5</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="r2">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>390</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="value">
     <number>5</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>460</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Ellipse Midpoint</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ell_polar">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>430</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Ellipse Polar</string>
    </property>
   </widget>
   <widget class="QLabel" name="exe_time_cir_polar">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>300</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="exe_time_cir_midpoint">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>330</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="exe_time_ell_polar">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>420</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="exe_time_ell_midpoint">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>460</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="exe_time_cir_cartesian">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>360</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="cir_cartesian">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>360</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Circle Cartesian</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>999</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>my_label</class>
   <extends>QLabel</extends>
   <header location="global">my_label.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
